Sources/HummingbirdAuth/Middleware/IsAuthenticatedMiddleware.swift

```diff
@@ -15,11 +15,13 @@
 import Hummingbird
 
 /// Middleware returning 401 for unauthenticated requests
-public struct IsAuthenticatedMiddleware<Auth: Authenticatable, Context: AuthRequestContext>: RouterMiddleware {
+public struct IsAuthenticatedMiddleware<Auth: Sendable, Context: AuthRequestContext>: RouterMiddleware {
     public init(_: Auth.Type) {}
```

Comment:
Given the identity type is already defined by the context this probably only needs a default init()